### PR TITLE
chore: add Python 3.13 support

### PR DESCRIPTION
**Pull request by gruebel**

### CODE REVIEW

**Comment by gruebel** [comment]
ok, this needs to wait for gRPC to support 3.13 https://github.com/grpc/grpc/issues/34922

**Comment by beeme1mr** [comment]
@gruebel, it looks like gRPC now supports Python 3.13!

**Comment by gruebel** [comment]
Nice, giving it a try.

**Review by federicobond — APPROVED** [review]

**Comment by federicobond** [comment]
Will this work without `allow-prereleases: true` now?

**Comment by gruebel** [comment]
Yeah, it should work without now.

**Review by beeme1mr — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           cache: "pip"
+          allow-prereleases: true
```
Comment (beeme1mr) [inline .github/workflows/build.yml:40]:
```suggestion
```

Let's see if we can remove this.

**Pull request merged**